core/modules.rs

```diff
@@ -30,7 +30,7 @@ use std::sync::Mutex;
 pub type SourceCodeInfoFuture =
   dyn Future<Item = SourceCodeInfo, Error = ErrBox> + Send;
 
-pub trait Loader: Send + Sync {
+pub trait Resolver: Send + Sync {
```

Comment:
Are these changes necessary? I'd rather have core changes done in a separate PR so they can be considered independently.

Reply:
I believe so, please check [this comment](https://github.com/denoland/deno/pull/3242#discussion_r341152908)